source/tutorials/php.rst

```diff
@@ -1,193 +1,251 @@
 .. _php:
 
 PHP and PHP-FPM
-***************
+###############
 
-This tutorial describes how to configure and use PHP and PHP-FPM  on |CL-ATTR|. Although we specifically address PHP this tutorial can serve as a general guide for working with applications in the |CL| :ref:`stateless` environment.
+This tutorial describes how to configure and use :abbr:`PHP (PHP: Hypertext Preprocessor)`
+and :abbr:`PHP-FPM (PHP FastCGI Process Manager)` on |CL-ATTR|.
 
 .. contents::
     :local:
     :depth: 1
 
-Overview
-********
+Description
+***********
 
-`PHP (PHP:Hypertext Preprocessor)`_ is an Open Source general-purpose scripting language that is popular with web-developers who leverage its ability to create dynamically generated web pages.  `PHP-FPM (FastCGI Process Manager)`_ is a PHP FastCGI implementation that controls process management, workers, and logging for PHP. The two applications work in conjunction but each has its own configuration.
+The `PHP language`_ is an Open Source general-purpose scripting language that is popular
+with web-developers who leverage its ability to create dynamically generated
+web pages. `PHP-FPM`_ is a PHP FastCGI implementation that controls process
+management, workers, and logging for PHP. The two applications work together,
+but each one has its own configuration.
 
+This tutorial specifically addresses PHP and PHP-FPM, however, it provides a
+general guide for working with applications in the |CL| :ref:`stateless`
+environment.
 
 
 Prerequisites
 *************
 
-* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system
-* Use :command:`swupd` to install the :file:`php-basic` bundle.
+* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system.
+* Use :command:`swupd` to install the :file:`php-basic` bundle:
 
   .. code-block:: bash
 
      sudo swupd bundle-add php-basic
 
 .. note::
 
-   PHP does not require a web server for operation.  Refer to :ref:`web-server-install` for instructions on setting up a :abbr:`LAMP (Linux, Apache\*, MySQL, PHP)` server, or use :command:`swupd` to install :file:`nginx` or similar if you need a web server.
+   PHP does not require a web server for operation. If you need a web
+   server, refer to :ref:`web-server-install` for instructions on setting
+   up a :abbr:`LAMP (Linux, Apache\*, MySQL, PHP)` server, or use
+   :command:`swupd` to install :file:`nginx` or similar.
 
 
 Configure PHP
 *************
 
 .. important::
 
-   This section does not cover configuring the PHP-FPM service. There is a difference between the two configurations. This section is only looking at the PHP configuration directives, not those of PHP-FPM.
+   This section does **not** describe configuration for the PHP-FPM service, which is
+   described later in this guide.
 
+This section creates a :file:`/etc/php.d` directory for all PHP configuration
```

Comment:
I had more explanation in this paragraph because I want to reinforce how to use stateless.  Shortening it to: "work around the default path restriction" leave people with no context about why CL needs to work around the defaults. I know the concise version reads better but it doesn't explain why we need a PHP tutorial in the first place, which is the intent for this doc.

Reply:
OK. Thanks for explaining your approach. Let's go with it.